packages/odyssey-react/src/components/Title/Title.tsx

```diff
@@ -10,28 +10,23 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import type {
-  ComponentPropsWithoutRef,
-  FunctionComponent,
-  ReactText,
-} from "react";
-import { useCx, useOmit } from "../../utils";
+import type { ComponentPropsWithoutRef, ReactText } from "react";
+import { forwardRef } from "react";
+import { useCx, useOmit, withStyles } from "../../utils";
 import styles from "./Title.module.scss";
 
-type Levels = "1" | "2" | "3" | "4" | "5" | "6";
-
-export interface Props
+interface Props
   extends Omit<ComponentPropsWithoutRef<"h1">, "style" | "className"> {
```

Comment:
Switch to `ComponentPropsWithRef` now that we have forward ref?

Reply:
@andrewberg-okta the other way around in fact. Even without a ref favor `ComponentPropsWithoutRef` in all cases unless there is some other critical extending circumstance!

Reply:
Let me rephrase, we don't include them in the interface?